README.md

```diff
@@ -1879,6 +1880,162 @@ FrontendWebsite | fe0b6a3   | ProductService | 1.0.0     | true
 All required verification results are published and successful
 ```
 
+## Step 12 - Using a PactFlow Broker
 
 
-That's it - you're now a Pact pro. Go build 🔨
+In step 11 we've been publishing our pacts from the consumer and provider projects to our locally hosted open source Pact broker.
+
+We can use a managed [Pact Broker](https://pactflow.io) over at PactFlow to do this instead.
+
+Using a hosted pact broker with pactflow, will allow you to concentrate on testing your application without the worry of additional infrastructure and neccessity to host the open source broker.
+
+### Creating a pactflow account
+
+Go to [Pact Flow](https://pactflow.io) and sign up for a new free account, you will be emailed some credentials.
+
+Use these to login to your pactflow broker.
+Click on settings -> API Tokens -> Read/write token -> COPY ENV VARS
+
+You can set these in your terminal
+
+```sh
+export PACT_BROKER_URL=https://<your_broker_name>.pact.dius.com.au
+export PACT_BROKER_TOKEN=exampleToken
+```
+
+
+```
+
+### Update your scripts to use the pact broker token based authentication method
+
+First, in the consumer project we need to tell Pact about our broker.
+
+In `consumer/publish.pact.js`:
+
+```groovy
```

Comment:
minor, but should this be `js` ?

Reply:
was copy-pasta from the previous step, but will update 👍🏾